src/components/dashboard/FeaturedApps/FeaturedApps.tsx

```diff
@@ -7,7 +7,7 @@ import NextLink from 'next/link'
 import { AppRoutes } from '@/config/routes'
 import { SafeAppsTag } from '@/config/constants'
 import { useRemoteSafeApps } from '@/hooks/safe-apps/useRemoteSafeApps'
-import SafeAppIcon from '@/components/safe-apps/SafeAppIcon'
+import SafeAppIconCard from '@/components/safe-apps/SafeAppIconCard/SafeAppIconCard'
```

Comment:
why the tautology? `SafeAppIconCard/SafeAppIconCard`

Reply:
it is just because `SafeAppIconCard` component is inside the `SafeAppIconCard` folder.

Reply:
I get why it happens, My question is related to why not follow how other components are stored `folder name/index.tsx` which allows you to only specify the folder name in the import statement

Reply:
done!